packages/experiment-browser/src/experiment-analytics-plugin.ts

```diff
@@ -0,0 +1,28 @@
+import { AmplitudeBrowser, Types } from '@amplitude/analytics-browser';
+
+import { ExperimentClient } from './experimentClient';
+import { initializeWithAmplitudeAnalytics } from './factory';
+
+// Module augmentation to add experiment variable at compile time
+declare module '@amplitude/analytics-browser' {
+  interface AmplitudeBrowser {
+    experiment: ExperimentClient;
+  }
+}
+
+export class ExperimentAnalyticsPlugin implements Types.EnrichmentPlugin {
+  name = 'experiment-analytics-plugin';
+  experiment: ExperimentClient;
+
+  constructor() {
+    Object.defineProperty(AmplitudeBrowser.prototype, 'experiment', {
+      get: function () {
+        return this.plugin(ExperimentAnalyticsPlugin.name);
```

Comment:
any reason to return the plugin vs the plugin's instance of experiment?

Reply:
I fixed it at https://github.com/amplitude/experiment-js-client/pull/168/commits/dee63ff78ef00db5c6ab8274367db1be1d7af016